build.sbt

```diff
@@ -220,6 +220,54 @@ lazy val `example-routeguide-client` = project
   .settings(addCommandAlias("runClientTask", "runMain example.routeguide.client.task.ClientAppTask"))
   .disablePlugins(ScriptedPlugin)
 
+////////////////////
+////  TODOLIST  ////
+////////////////////
+
+lazy val `example-todolist-protocol` = project
+  .in(file("modules/examples/todolist/protocol"))
+  .dependsOn(client)
+  .settings(moduleName := "frees-rpc-example-todolist-protocol")
+  .disablePlugins(ScriptedPlugin)
+
+lazy val `example-todolist-runtime` = project
+  .in(file("modules/examples/todolist/runtime"))
+  .settings(moduleName := "frees-rpc-example-todolist-runtime")
+  .settings(exampleTodolistRuntimeSettings)
+  .disablePlugins(ScriptedPlugin)
+
+lazy val `example-todolist-common` = project
+  .in(file("modules/examples/todolist/common"))
+  .dependsOn(`example-todolist-protocol`)
+  .dependsOn(config)
+  .settings(moduleName := "frees-rpc-example-todolist-common")
+  .settings(exampleTodolistCommonSettings)
+  .disablePlugins(ScriptedPlugin)
+
+lazy val `example-todolist-server` = project
+  .in(file("modules/examples/todolist/server"))
+  .dependsOn(`example-todolist-common`)
+  .dependsOn(`example-todolist-runtime`)
+  .dependsOn(server)
+  .settings(moduleName := "frees-rpc-example-todolist-server")
+  .disablePlugins(ScriptedPlugin)
+
+lazy val `example-todolist-client` = project
+  .in(file("modules/examples/todolist/client"))
+  .dependsOn(`example-todolist-common`)
+  .dependsOn(`example-todolist-runtime`)
+  .dependsOn(`client-netty`)
+  .settings(moduleName := "frees-rpc-example-todolist-client")
+  .settings(
+    Compile / unmanagedSourceDirectories ++= Seq(
+      baseDirectory.value / "src" / "main" / "scala-io",
+      baseDirectory.value / "src" / "main" / "scala-task"
```

Comment:
I'd suggest removing this double interpretation in this example, keeping only the IO example.

Reply:
:+1: